activejob/test/cases/job_serialization_test.rb

```diff
@@ -61,4 +61,10 @@ class JobSerializationTest < ActiveSupport::TestCase
       assert_equal "Hawaii", job.serialize["timezone"]
     end
   end
+
+  test "serialize stores the enqueued_at time" do
+    job = HelloJob.new
+    type = job.serialize["enqueued_at"].class
+    assert_equal Time, type
```

Comment:
Not sure if returning a Time object from serialize is a good idea. It will work in JSON-based adapters but the problem is that the roundtrip of this value will return a String, so maybe it better to be explicit about the string conversion we want to do?

Reply:
Is there a Rails preferred format for this sort of data?

Reply:
I changed it to be a string then added a test to make sure it could be parsed.